server/athenian/api/align/queries/metrics.py

```diff
@@ -386,11 +413,11 @@ async def calculate_team_metrics(
 
     team_metrics_res: TeamMetricsResult = {}
     if pr_requests:
-        pr_collector.collect(calc_results.pop(0), team_metrics_res)
+        team_metrics_res.update(pr_collector.collect(calc_results.pop(0)))
     if release_requests:
-        release_collector.collect(calc_results.pop(0), team_metrics_res)
+        team_metrics_res.update(release_collector.collect(calc_results.pop(0)))
     if jira_requests:
-        jira_collector.collect(calc_results[0], team_metrics_res)
+        team_metrics_res.update(jira_collector.collect(calc_results[0]))
 
```

Comment:
Instead of calling one function, we now do 2x more operations and increase the cognitive load + the code complexity. It is better to return back the old code. Simpler >>> complex.

Reply:
`collect` now is a pure function and this should be usually perceived as a huge code simplification. reverting.